yt_dlp/extractor/youtube.py

```diff
@@ -2287,228 +2302,122 @@ def _get_video_info_params(video_id, client='TVHTML5'):
-                        # Unquote should take place before split on comma (,) since textual
-                        # fields may contain comma as well (see
-                        # https://github.com/ytdl-org/youtube-dl/issues/8536)
-                        feed_data = compat_parse_qs(
-                            compat_urllib_parse_unquote_plus(feed))
-
-                        def feed_entry(name):
-                            return try_get(
-                                feed_data, lambda x: x[name][0], compat_str)
+    def _extract_player_responses(self, clients, video_id, webpage, master_ytcfg, player_url, identity_token):
+        if webpage:
+            initial_pr = self._extract_yt_initial_variable(
+                webpage, self._YT_INITIAL_PLAYER_RESPONSE_RE,
+                video_id, 'initial player response')
 
-                        feed_id = feed_entry('id')
-                        if not feed_id:
-                            continue
-                        feed_title = feed_entry('title')
-                        title = video_title
-                        if feed_title:
-                            title += ' (%s)' % feed_title
-                        entries.append({
-                            '_type': 'url_transparent',
-                            'ie_key': 'Youtube',
-                            'url': smuggle_url(
-                                base_url + 'watch?v=' + feed_data['id'][0],
-                                {'force_singlefeed': True}),
-                            'title': title,
-                        })
-                        feed_ids.append(feed_id)
-                    self.to_screen(
-                        'Downloading multifeed video (%s) - add --no-playlist to just download video %s'
-                        % (', '.join(feed_ids), video_id))
-                    return self.playlist_result(
-                        entries, video_id, video_title, video_description)
+        for client in clients:
+            ytpcfg = master_ytcfg if client == 'web' else {}
+            if client == 'web' and initial_pr:
+                pr = initial_pr
             else:
-                self.to_screen('Downloading just video %s because of --no-playlist' % video_id)
-
-        formats, itags, stream_ids = [], [], []
+                if client == 'web_music' and 'configs' not in self._configuration_arg('player_skip'):
+                    ytm_webpage = self._download_webpage(
+                        'https://music.youtube.com',
+                        video_id, fatal=False, note='Downloading remix client config')
+                    ytpcfg = self._extract_ytcfg(video_id, ytm_webpage) or {}
+                pr = self._extract_player_response(
+                    client, video_id, ytpcfg or master_ytcfg, ytpcfg, identity_token, player_url, initial_pr)
+            if pr:
+                yield pr
+            if traverse_obj(pr, ('playabilityStatus', 'reason')) in self._AGE_GATE_REASONS:
```

Comment:
we should make note of when we have detected the video is age-gated so we don't bother extracting any more non-age gate pr's from additional clients.